backend/api/Areas/Admin/Controllers/UserController.cs

```diff
@@ -93,12 +93,6 @@ public IActionResult GetUsers(EModel.UserFilter filter)
         [SwaggerOperation(Tags = new[] { "admin-user" })]
         public IActionResult GetMyUsers(EModel.UserFilter filter)
         {
-            if (!(filter?.Agencies?.Any() ?? false))
-            {
-                filter.Agencies = this.User.GetAgencies();
```

Comment:
Does this mean that if the filter contains agencies it will use those instead of the users agencies?  If so that would provide an **Agency Administrator** a way to view and edit users not in their agency.  This isn't ideal.  Only **System Administrators** can view and edit all users.

This endpoint should always and only return users that the current user is allowed to view and edit.

Reply:
The agency id filter is applied in the users services. See changes in the Get users